proto/topic_message.proto

```diff
@@ -5,15 +5,9 @@ package proto;
 option go_package = "github.com/limechain/hedera-eth-bridge-validator/proto";
 
 import "topic_eth_signature_message.proto";
-import "topic_eth_transaction_message.proto";
-import "topic_message_type.proto";
 
 // TopicMessage
 message TopicMessage {
-  TopicMessageType type = 1;
-  oneof message {
-    TopicEthSignatureMessage topicSignatureMessage = 2;
-    TopicEthTransactionMessage topicEthTransactionMessage = 3;
-  }
-  int64 transactionTimestamp = 4;
+  TopicEthSignatureMessage topicSignatureMessage = 1;
+  int64 transactionTimestamp = 2;
```

Comment:
Is there a reason we are having the `transactionTimestamp` in the Topic message? It seems that we are not even populating it when we instansiate new `TopicMessage` in `model/message/message.go`. If that is the case, we could directly use only one proto "message" -> `TopicMessage` that contains the data in the `TopicEthSignatureMessage`